apps/music/js/ui/views/mode_manager.js

```diff
@@ -107,21 +107,27 @@ var ModeManager = {
     if (mode === MODE_PLAYER) {
       // Here if Player is not loaded yet and we are going to play
       // load player_view.js then we can use the PlayerView object
-      document.getElementById('views-player').classList.remove('hidden');
       LazyLoader.load('js/ui/views/player_view.js', function() {
         if (!playerLoaded) {
           PlayerView.init();
           PlayerView.setOptions(App.playerSettings);
         }
-
         // Music only share the playing file when it's in player mode.
         this._enableNFCSharing(true);
 
         if (callback) {
           callback();
         }
+        document.getElementById('views-player').classList.remove('hidden');
```

Comment:
Good call to move this line inside the lazy load. Note that by putting it here after the callback you're assuming that the callback will never throw an exception.  A more defensive coding style would have you use a try/finally block.  Or just show the element right before calling the callback...